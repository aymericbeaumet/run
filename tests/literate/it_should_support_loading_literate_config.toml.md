This is a literate configuration in Markdown!

```toml
[[run]]
cmd = ["coreutils", "echo", "one"]
```

```toml
[[run]]
cmd = ["coreutils", "echo", "two"]
```

```toml
[[run]]
cmd = ["coreutils", "echo", "three"]
```