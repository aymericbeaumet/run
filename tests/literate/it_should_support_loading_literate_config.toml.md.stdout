[coreutils] one
[coreutils] two
[coreutils] three